pkg/ingestor/parser/dsse/parser_dsse.go

```diff
@@ -69,7 +68,7 @@ func (d *dsseParser) getIdentity(ctx context.Context) error {
 			logger := logging.FromContext(ctx)
 			logger.Errorf("failed to verify DSSE with provided key: %w", i.ID)
 		}
-	}
+	} */
```

Comment:
Shall we log a warning here so that users don't assume dsses are validated?

Reply:
Added warning message. PTAL